web-app/js/portal/filter/BooleanFilterPanel.js

```diff
@@ -41,8 +41,7 @@ Portal.filter.BooleanFilterPanel = Ext.extend(Portal.filter.BaseFilterPanel, {
     },
 
     _formatBoxLabel: function() {
-        var label = this.filter.label.split('_').join(' ').toTitleCase();
-        return label;
+        return this.filter.getDisplayLabel();
```

Comment:
... or serialises a filter in to a URL key/value param - so that we can then construct links which includes filters....